purchase_variant_configurator_on_confirm/models/purchase_order_line.py

```diff
@@ -18,22 +19,19 @@ class PurchaseOrderLine(models.Model):
 
     @api.onchange('product_tmpl_id')
     def onchange_product_tmpl_id(self):
-        # this method is copied from the standard onchange_product_id
+        """ Copy from standard: `PurcharseOrderLine.onchange_product_id` """
```

Comment:
Have you reviewed v10 standard methods and their code? Maybe you can use similar techniques to the one I suggested above.